helm-chart/renku/requirements.yaml

```diff
@@ -36,3 +36,8 @@ dependencies:
     repository: "oci://registry-1.docker.io/bitnamicharts"
     version: "8.9.2"
     condition: solr.enabled
+  - name: common
```

Comment:
Just a reminder to add a comment here to explain why this is pinned here.

Reply:
done